### ISSUE TITLE
ixpeobssim not compatible with matplotlib 3.6.0

### ISSUE THREAD

**Issue by ndilalla** [lucabaldini/ixpeobssim#631]
I was trying to test the package locally in a brand new environment and I see that several tests are failing due to the same error:
```
ValueError: Unable to determine Axes to steal space for Colorbar. Either provide the *cax* argument to use as the Axes for the Colorbar, provide the *ax* argument to steal space from it, or add *mappable* to an Axes.

../../../miniconda3/envs/test_ixpeobssim/lib/python3.9/site-packages/matplotlib/figure.py:1256: ValueError
```
Searching online looks like something changed with the colorbar in the new version of ``matplotlib`` released a week ago. No problems using ``matplotlib 3.5.3``.

**Comment by lucabaldini** [comment]
A couple useful pointers:
https://github.com/matplotlib/matplotlib/issues/23709
https://github.com/matplotlib/matplotlib/pull/23740

@albertomanfreda Can you take a look?

**Comment by albertomanfreda** [comment]
I I am on it

**Comment by lucabaldini** [comment]
Thanks Alberto---I am testing the change and merging the pull request right now.

Note that you have push priviledges on the main repo, now, and you can open a branch directly here, rather than working on a fork.

**Comment by lucabaldini** [comment]
Shipped in version 29.1.4
